File: src/Mercu.Kanban.WebApi/Mercu.Kanban.Application/DTOS/CandidateModel.cs
namespace Mercu.Kanban.Application.DTOS
{
#pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
    public record CandidateModel : CandidateModelBase
    {
        public IList<JobModel> JobModels { get; set; }
        public IList<InterviewerModel> InterviewerModels { get; set; }
    }
}

File: src/UI/src/app/components/models/candidate.model.ts
import { InterviewerModel,JobModel } from "./index";

export interface CandidateModel {
    id: number,
    firstName: string,
    lastName: string,
    phoneNumber: string,
    email: string,
    createDate: string,
    jobModels: Array<JobModel>,
    interviewerModels : Array<InterviewerModel>
}